contracts/modules/Swap.sol

```diff
@@ -472,4 +597,19 @@ contract Swap is BaseLogic {
             result := div(mload(add(add(data, 0x20), start)), 0x1000000000000000000000000)
         }
     }
+
+    function getAggregatedSwapResult(bytes memory result) private pure returns (uint) {
```

Comment:
Can we get a comment on what this function is doing? Which methods are returning `uint` vs `uint[]`?